scope & closures/ch2.md

```diff
@@ -72,27 +72,27 @@ Não importa o *local* onde uma função é invocada, ou até mesmo *como* é in
 
 O processo de consulta ao escopo léxico ocorre *apenas" em identificadores de primeira classe, como `a`, `b` e `c`. Se você tivesse uma referência para `foo.bar.baz` em um trecho de código, ocorreria uma consulta ao escopo léxico para localizar o identificador `foo`, mas no momento que esta variável é localizada, regras de acesso à propriedades de objetos assumem o comando para resolução das propriedades `bar` e `baz`, respectivamente.
 
-## Cheating Lexical
+## Trapaceando o Léxico
 
-If lexical scope is defined only by where a function is declared, which is entirely an author-time decision, how could there possibly be a way to "modify" (aka, cheat) lexical scope at run-time?
+Se o escopo léxico é de fato definido apenas pelo local onde uma função é declarada, cuja decisão vem do autor do código durante a programação, como pode haver uma maneira de "modificar" (ou trapacear) o escopo léxico em tempo de execução?
```

Comment:
~~cuja decisão vem do autor do código durante a programação~~ **que é uma decisão inteiramente _author-time_**

`Author-time` se refere ao momento de escrita (autoria) do escopo (léxico), não o tempo que ele é executado (runtime). Foi essa associação que ele quis dizer. 

O exemplo que consigo pensar agora é *this*, que tem sua associação (*binding*) executada no *runtime*, não em *author-time*. Ou seja, onde você escreve ele não importa, no momento da execução do código (runtime), o JS define onde o `this` está.

Coloquei o exemplo porque também ainda não pensei num termo apropriado para *author-time*, se tiver alguma sugestão será ótimo. Se de tudo não conseguirmos chegar num modelo ideal, acho válido manter *author-time*. Ele explica mais sobre *author-time/runtime* no próximo livro, mas não li com cuidado a ponto de opinar.